src/utils.rs

```diff
@@ -1,22 +1,18 @@
-#[derive(Debug)]
-pub struct RGB {
-    red: u8,
-    green: u8,
-    blue: u8,
+pub fn u1u5u5u5_to_u5u5u5u1(u1u5u5u5: u16) -> u16 {
```

Comment:
What about renaming to `gba_to_display_format`? The u1u5u5u5 seems very confusing.